### PR TITLE
Cockpit 259

### PR DESCRIPTION
**Pull request by garrett**
Initial draft

Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-259.html

### CODE REVIEW

**Comment by garrett** [comment]
I'm unable to update the release notes right now due to an unspecified GitHub API error. :shrug:

**Comment by mvollmer** [comment]
I have added cockpit-machines and cockpit-podman and removed unmerged PRs.

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]
I think a mention of cockpit client is worthwhile.

## Cockpit Client

Cockpit now ships [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient) via [Flatpak](https://flatpak.org/).

If you are on a Linux desktop, this is now the most convenient and secure way (no open ports other than ssh) of accessing remote servers or your local machine with Cockpit.

**Comment by mvollmer** [comment]
> I think a mention of cockpit client is worthwhile.

Thanks, included!

File: _posts/2021-12-08-cockpit-259.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 259, cockpit-machines 258, and cockpit-podman 38:
+
+## Storage: More information in table rows
+
+Table rows representing filesystems now show more information. You can
+now immediately see the mountpoint, the usage, and there are also
```
Comment (allisonkarlitskaya) [inline _posts/2021-12-08-cockpit-259.md:19]:
```suggestion
now immediately see the mountpoint and the usage, and there are also
```

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]

File: _posts/2021-12-08-cockpit-259.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+links to the pages for volume groups.
+
+![screenshot of more information in table rows]({{ site.baseurl }}/images/259-storage-more-information-in-table-rows.png)
+
+## Cockpit Client
+
+Cockpit now ships [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient) via [Flatpak](https://flatpak.org/).
+
+If you are on a Linux desktop, this is now the most convenient and secure way (no open ports other than ssh) of accessing remote servers or your local machine with Cockpit.
```
Comment (martinpitt) [inline _posts/2021-12-08-cockpit-259.md:28]: Does a screenshot make sense? We could take it from https://flathub.org/apps/details/org.cockpit_project.CockpitClient

File: _posts/2021-12-08-cockpit-259.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Try it out
+
+Cockpit 259 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/259)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit)
```
Comment (martinpitt) [inline _posts/2021-12-08-cockpit-259.md:37]: All bodhi URLs for c, machines, and podman are available since Wednesday, please update them here.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks!

File: _posts/2021-12-08-cockpit-259.md (hunk 3 above)
Comment (mvollmer) [reply _posts/2021-12-08-cockpit-259.md]: Done.

File: _posts/2021-12-08-cockpit-259.md (hunk 2 above)
Comment (mvollmer) [reply _posts/2021-12-08-cockpit-259.md:28]: I don't know, it's just a screenshot of Cockpit...

**Review by martinpitt — APPROVED** [review]
Danke!

File: _posts/2021-12-08-cockpit-259.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+title: Cockpit 259
+author: garrett
+date: '2021-12-08'
+tags: cockpit
+slug: cockpit-259
+category: release
+summary: ''
```
Comment (martinpitt) [inline _posts/2021-12-08-cockpit-259.md:8]: Whoops! That one is still missing

**Review by martinpitt — CHANGES_REQUESTED** [review]

**Review by martinpitt — APPROVED** [review]

**Pull request merged**